module/apmgopgv10/go.mod

```diff
@@ -0,0 +1,14 @@
+module go.elastic.co/apm/module/apmgopg
```

Comment:
```suggestion
module go.elastic.co/apm/module/apmgopgv10
```

Sorry, missed this one before! This should fix the CI failures.

Reply:
Ah, my bad I missed it too.

Reply:
Didn't seam to help